frontend/src/modules/entitydetails/components/Metadata.test.js

```diff
@@ -47,7 +47,7 @@ describe('<Metadata>', () => {
         const content = wrapper.find('Linkify').map(item => item.props().children);
         expect(content).toContain(ENTITY.comment);
 
-        expect(wrapper.find('#entitydetails-Metadata--resource a').text()).toContain(ENTITY.path);
+        expect(wrapper.find('#entitydetails-Metadata--resource a').at(1).text()).toContain(ENTITY.path);
```

Comment:
Let's append `.resource-path` to the selector instead of using `at()` - same reason as for avoiding `nth-child`.